guestbook/api/api_view.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+from django.views.generic import TemplateView
+from guestbook.api import jsonresponse
+from guestbook.models import Greeting, greeting_to_obj, greeting_to_dict
+
+
+class GreetingList(jsonresponse.JSONResponseMixin, TemplateView):
+
+	def get_context_data(self, **kwargs):
+		urlsafe = self.request.GET.get('urlsafe', '')
+		guestbook_name = kwargs['guestbook_name']
+		greetings, next_urlsafe, more = Greeting.get_greeting_by_page(guestbook_name, urlsafe)
```

Comment:
Hiện tại hardcode number of page là 5.
Chỉnh hàm này 1 chút: `get_greeting_by_page(cls, guestbook_name, limit, cursor, **kwargs)`